utils/Money/Money.tsx

```diff
@@ -0,0 +1,154 @@
+import { MoneyStatic, MoneyStaticUSD } from "utils/Money/money.static";
+
+import { components } from "src/__generated/api";
+
+interface IMoneyClassFormat {
+  amount?: number | null;
+  currency?: Money.Currency;
+  options?: {
+    showCurrency?: boolean;
+    locale?: string;
+    currencyClassName?: string;
+    prefixAmountWithTilde?: boolean;
+    notation?: "standard" | "scientific" | "engineering" | "compact";
+  };
+}
+
+interface IMoneyClass {
+  format: (props: IMoneyClassFormat) => { string: string; html: JSX.Element };
+  isFiat: (currency?: Money.Currency) => boolean;
+  fromSchema: (props: Partial<Money.Currency>) => Money.Currency;
+}
+
+class MoneyClass implements IMoneyClass {
+  private CodeMapping = {
+    FIAT: ["USD", "EUR"],
+  };
+  private withCurrency(str: string, code: string, showCurrency: boolean) {
+    if (showCurrency) {
+      return `${str} ${code}`;
+    }
+
+    return str;
+  }
+
+  private withTilde(str: string | number, prefixAmountWithTilde: boolean) {
```

Comment:
Isn't the amount always a string here ?

```suggestion
  private withTilde(str: string, prefixAmountWithTilde: boolean) {
```